edsl.md

```diff
@@ -182,6 +184,9 @@ where `F1 : F2 : F3 : F4` is the (two's complement) byte-array representation of
     rule #buf(SIZE, DATA) => #padToWidth(SIZE, #asByteStack(DATA))
       requires #range(0 <= DATA < (2 ^Int (SIZE *Int 8)))
       [concrete]
+    //todo prohibitive performance drop for ERC20 verifier when full rule is used.
+    //rule #Ceil(#buf(SIZE, DATA)) => {(0 <=Int SIZE) andBool #range(0 <= DATA < (2 ^Int (SIZE *Int 8))) #Equals true}  [anywhere]
+    rule #Ceil(#buf(SIZE, DATA)) => {(0 <=Int SIZE) #Equals true}  [anywhere]
```

Comment:
I would rather not assert false things here if possible.

Reply:
What's your solution then?

Reply:
I don't have one, but I don't really want to merge this into KEVM, because then other proofs may depend on it. How big is the slowdown?

Maybe if this isn't fixable, then we should just not merge this now, and consider it for the next "controversial changes" PR.